src/Symfony/Component/Translation/DependencyInjection/TranslatorPass.php

```diff
@@ -19,13 +19,17 @@
 class TranslatorPass implements CompilerPassInterface
 {
     private $translatorServiceId;
-    private $loaderServiceId;
+    private $readerServiceId;
     private $loaderTag;
 
-    public function __construct($translatorServiceId = 'translator.default', $loaderServiceId = 'translation.loader', $loaderTag = 'translation.loader')
+    public function __construct($translatorServiceId = 'translator.default', $readerServiceId = 'translation.loader', $loaderTag = 'translation.loader')
     {
+        if ('translation.loader' === $readerServiceId) {
+            @trigger_error('The default value for $readerServiceId will change in 4.0 to "translation.reader".', E_USER_DEPRECATED);
```

Comment:
How should I handle all the tests that are failing because of this deprecation notice?